aspen/wsgi.py

```diff
@@ -18,3 +18,4 @@
 from aspen.website import Website
 
 website = Website([])
+application = website  # many servers look for this name
```

Comment:
I don't understand your comment. Are you referring to the word "servers" in comment in this line? What's your predicate?